membership-attribute-service/app/models/UserFromToken.scala

```diff
@@ -61,6 +64,22 @@ object UserFromToken {
       firstName = user.privateFields.firstName,
       lastName = user.privateFields.secondName,
       authTime = None,
+      oktaId = "", // no okta id in idapi, it's not used by MDAPI
     )
+  }
+
+}
+case class MDAPIIdentityClaims(oktaId: String, primaryEmailAddress: String, identityId: String) extends IdentityClaims
+
+object MDAPIIdentityClaims {
+
+  implicit val identityClaimsParser: UserInfoResponseParser[MDAPIIdentityClaims] =
+    (rawClaims: JsonString) => {
+      for {
+        oktaId <- UserInfoResponseParser.oktaId(rawClaims)
+        primaryEmailAddress <- UserInfoResponseParser.primaryEmailAddress(rawClaims)
+        identityId <- UserInfoResponseParser.identityId(rawClaims)
+      } yield MDAPIIdentityClaims(oktaId, primaryEmailAddress, identityId)
+    }
```

Comment:
`UserInfoResponseParser` seemed to be something we didn't have to specify before, but we do now.  We don't actually need them in MDAPI but I had to make one to keep validateCredentialsFromRequest happy